80-hour embargo period in survey-mode LSSTCam commissioning (#26)
lsst-dm/dmtn-199: intro.tex

\item	Encrypt data using strong, approved encryption standards,  following \citeds{NIST.SP.800-171} for \gls{CUI} at non federal organizations. The required use of these \gls{NIST} security standards is limited to the physical security and encryption points. It does not extend to treating the data as \gls{CUI}. The data should \emph{not} be marked as \gls{CUI}.
\item	Use firewalls and physical security best practices to prevent unauthorized network  access. Documented compliance shall be in accordance with \citeds{NIST.SP.800-171}.
\item	Delay public release of  focal plane scientific data for an embargo period of at least 80 hours following the  observation.

> Hold engineering-mode commissioning imaging data for an embargo period of at least 30 days.

Reviewer: that changes the meaning - this says i can publish on sky images which are not engineering mode ...

Reviewer 2: Maybe "Hold engineering mode images for 30 days and science validation images for regular operations period of 80 hours. 

This could be the regular mode for all of ops, where we modify validation to just regular on sky or some such.

Author: Adopted @rblum5 's formulation, with some tweaks

> Hold engineering-mode commissioning imaging data for an embargo period of at least 30 days.

Author: ```suggestion
Hold engineering mode images for 30 days and survey-mode images (e.g. Science Validation survey images) for the regular survey operations period of 80 hours.
```

In the rare event an unknown solar system object source with a streak length greater than 10 \gls{deg}/day relative to sidereal potentially corresponds with an Earth-impacting asteroid, the \gls{USDF} will implement a method to send any candidate impactors to the \gls{MPC} during the embargo period.
All data in transit shall comply with the strong, approved encryption standards outlined for the embargo period.

> In addition, commissioning engineering data will be embargoed for all non-commissioning team staff for 30 days.

Author: ```suggestion
In addition, commissioning data taken prior to System First Light (SFL) will be embargoed for all non-commissioning team staff for 30 days.
```

After this 30 day embargo, only with explicit approval may proprietary data products from commissioning be shared outside the \gls{Commissioning} Team  \citeds{SITCOMTN-010}.

> The transition from the 30-day embargo to an 80-hour embargo will happen at System First Light (SFL), as long as all federal reviews of Rubin Data Management Standards have been passed. 

Author: ```suggestion
The transition from the 30-day embargo to an 80-hour embargo will happen at System First Light (SFL), and when all federal reviews of Rubin Data Management Standards have been passed. 
```

> The transition from the 30-day embargo to an 80-hour embargo will happen at System First Light (SFL), and when all federal reviews of Rubin Data Management Standards have been passed. 

Author: ```suggestion
The transition from the 30-day embargo to an 80-hour embargo will happen when all federal reviews of Rubin Data Management Standards have been passed. 
```

Author: Decouple embargo change from Rubin-determined SFL, and place it with agencies

> NSF and DOE will provide simple email confirmation of this, following the final review outbrief.

Author: ```suggestion
NSF and DOE will provide written confirmation of this.
```

> (Prior to SFL, LSSTCam commissioning takes place in ``engineering mode,'' with observations executed in custom blocks that do not resemble routine sky surveying. 

Author: ```suggestion
(Prior to SFL, LSSTCam commissioning takes place predominantly in ``engineering mode,'' with observations executed in custom blocks that do not resemble routine sky surveying. 
```

> After SFL, Rubin will switch to ``survey mode,'' with observations executed in schedule blocks similar to those of the LSST.)  

Author: ```suggestion
After SFL, Rubin will switch to ``survey mode,'' with most observations scheduled and executed in the same way as those of the LSST.)  
```
